src/service/communication.service.ts

```diff
@@ -35,38 +36,41 @@ export class BBHelpCommunicationService {
     });
   }
 
-  public messageHandler() {
+  public isFromHelpWidget(event: { origin: string, data: any }): boolean {
+    if (event.origin === HOST_ORIGIN) {
+      const message = event.data;
+      return !!message && message.source === 'skyux-spa-bb-help';
+    }
+
+    return false;
+  }
+
+  public postMessage(message: any, origin: string = HOST_ORIGIN) {
+    message.source = 'help-client';
+    this.childWindow.contentWindow.postMessage(message, origin);
+  }
+
+  private messageHandler() {
     return (event: any) => {
       if (this.isFromHelpWidget(event)) {
         const message = event.data;
         switch (message.messageType) {
           case 'ready':
             this.postMessage({ messageType: 'host-ready' });
-            this.communicationAction.next('Child Window Ready');
+            this.communicationAction.next({ messageType: 'Child Window Ready'});
             this.childWindowReady = true;
             break;
           case 'close-widget':
-            this.communicationAction.next('Close Widget');
+            this.communicationAction.next({ messageType: 'Close Widget'});
+            break;
+          case 'config-loaded':
+            this.communicationAction.next({ messageType: 'Config Loaded', data: message.data });
             break;
           default:
-            console.error(`No matching response for message type: ${message.messageType}`);
+            console.error(`No matching response for message type: ${ message.messageType }`);
```

Comment:
Linting catches this on other projects, but I'm not sure about how it's setup for this one.  For consistency I would remove the spaces around the variable, so `${message.messageType}`.

Reply:
Sounds good